commands/rss.py

```diff
@@ -0,0 +1,24 @@
+name = 'rss'
+help = '''%rss list | add <url> | del <url>
+Manage the list of feeds that bot will follow and post updates from.
+RSS and Atom are both supported. Only available for manager accounts.'''
+
+
+async def handler(args, request):
+    if request.event.sender in request.bot.cfg.manager_accounts:
+        if not len(args) or len(args) == 1 and args[0] == 'list':
+            urls = '\n'.join(list(request.bot.feeder.feeds.keys()))
+            await request.reply(f'feeds list:\n{urls}')
```

Comment:
I think rss list should be usable by unprivileged users as well

Reply:
Since it doesn't really matter for me, OK I'll do so.